src/libs/tools/global_toolbox.c

```diff
@@ -18,6 +18,7 @@
 
 #include "common/collection.h"
 #include "common/darktable.h"
+#include "common/debug.h"
```

Comment:
is this needed ? that's the only change in this file.

Reply:
oops, sorry... leftover from a first test... removed !